IntelliTect.TestTools.Console.Tests/ConsoleAssertTests.cs

```diff
@@ -51,6 +51,20 @@ public void ConsoleTester_HelloWorld_TrimLF()
             }, true);
         }
 
+        [TestMethod]
+        [DataRow(@"[49mMontoya", "Montoya")]
+        [DataRow("Inigo[49mMontoya", "InigoMontoya")]
+        [DataRow("Inigo[49m", "Inigo")]
+        [DataRow("[49m", "")]
+        [DataRow("[101mMontoya", "Montoya")]
+        public void ConsoleTester_StringWithVT100Characters_VT100Stripped(string input, string expected)
```

Comment:
I would add in a second test that ensures that if you pass false for `stripVT100` that they are left in.